### ISSUE TITLE
Provider org.apache.seatunnel.connectors.seatunnel.file.s3.sink.S3FileSink could not be instantiated

### ISSUE THREAD

**Issue by guoyangjie** [apache/seatunnel#6676]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

When we use the S3 protocol to synchronize CSV files from MinIO to ClickHouse, an exception is thrown. The source is S3, and the target is ClickHouse.

### SeaTunnel Version

2.3.4

### SeaTunnel Config

```conf
{
    "env": {
        "job.mode": "BATCH"
    },

    "source": [
        {
            "path": "/collect/test.csv",
            "bucket": "s3a://datamanager",
            "schema": {
                "fields": {
                    "ID": "int",
                    "NAME": "string"
                }
            },
            "access_key": "GMWCEab2XZNyINHb4nXM",
            "secret_key": "QPmEPsrZuacLiu8gq0zpUxe9QsetJGtrLvgytRWp",
            "plugin_name": "S3File",
            "read_columns": [
                "ID",
                "NAME"
            ],
            "fs.s3a.endpoint": "http://192.168.3.1:9000",
            "fs.s3a.aws.credentials.provider": "org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider",
            "file_format_type": "csv",
            "result_table_name": "sourceTable",
            "skip_header_row_number": 1
            
        }
    ],
    "transform": [
        {
            "query": "select ID as id,NAME as name from sourceTable  where 1=1 ",
            "plugin_name": "Sql",
            "result_table_name": "sinkTable",
            "source_table_name": "sourceTable"
        }
    ],
        "sink": [
        {
            "host": "192.168.3.1:8123",
            "table": "test_1",
            "database": "default",
            "password": "",
            "username": "default",
            "plugin_name": "Clickhouse",
            "source_table_name": "sinkTable"
        }
    ]
}
```

### Running Command

```shell
seatunnel.cmd --config test.conf
```

### Error Exception

```log
java.util.ServiceConfigurationError: org.apache.seatunnel.api.sink.SeaTunnelSink: Provider org.apache.seatunnel.connectors.seatunnel.file.s3.sink.S3FileSink could not be instantiated
        at java.util.ServiceLoader.fail(ServiceLoader.java:232) ~[?:1.8.0_382]
        at java.util.ServiceLoader.access$100(ServiceLoader.java:185) ~[?:1.8.0_382]
        at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384) ~[?:1.8.0_382]
        at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404) ~[?:1.8.0_382]
        at java.util.ServiceLoader$1.next(ServiceLoader.java:480) ~[?:1.8.0_382]
        at org.apache.seatunnel.plugin.discovery.AbstractPluginDiscovery.loadPluginInstance(AbstractPluginDiscovery.java:302) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.plugin.discovery.AbstractPluginDiscovery.createOptionalPluginInstance(AbstractPluginDiscovery.java:183) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.plugin.discovery.AbstractPluginDiscovery.createPluginInstance(AbstractPluginDiscovery.java:227) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.core.parse.ConnectorInstanceLoader.loadSinkInstance(ConnectorInstanceLoader.java:77) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.core.parse.JobConfigParser.parseSink(JobConfigParser.java:194) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.core.parse.JobConfigParser.parseSinks(JobConfigParser.java:170) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.core.parse.MultipleTableJobConfigParser.parseSink(MultipleTableJobConfigParser.java:531) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.core.parse.MultipleTableJobConfigParser.parse(MultipleTableJobConfigParser.java:193) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.engine.server.rest.RestJobExecutionEnvironment.getLogicalDag(RestJobExecutionEnvironment.java:76) ~[seatunnel-starter.jar:2.3.4]
        at org.apache.seatunnel.en
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by pppeace** [comment]
same phenomenon.

**Comment by Hisoka-X** [comment]
please retry with dev branch. Already fixed by #6551

**Comment by guoyangjie** [comment]
Thank you, I'll give it a try
